tests/test_parastell.py

```diff
@@ -29,28 +29,171 @@ def remove_files():
             Path.unlink(file)
 
 
+def check_cubit():
+    try:
+        import cubit
+
+        return True
+    except ImportError:
+        return False
+
+
 def check_surfaces_and_volumes(filename, num_surfaces_exp, num_volumes_exp):
     dagmc_model = dagmc.DAGModel(str(Path(filename).with_suffix(".h5m")))
 
     assert len(dagmc_model.surfaces) == num_surfaces_exp
     assert len(dagmc_model.volumes) == num_volumes_exp
 
 
+def create_ivb_cad_magnets_from_filaments(stellarator_obj):
+    toroidal_angles = [0.0, 5.0, 10.0, 15.0]
+    poloidal_angles = [0.0, 120.0, 240.0, 360.0]
+    wall_s = 1.08
+    component_name_exp = "component"
+    radial_build_dict = {
+        component_name_exp: {
+            "thickness_matrix": np.ones(
+                (len(toroidal_angles), len(poloidal_angles))
+            )
+            * 10
+        }
+    }
+    num_ribs = 11
+
+    stellarator_obj.construct_invessel_build(
+        toroidal_angles,
+        poloidal_angles,
+        wall_s,
+        radial_build_dict,
+        num_ribs=num_ribs,
+    )
+
+    stellarator_obj.export_invessel_build()
+
+    coils_file = Path("files_for_tests") / "coils.example"
+    width = 40.0
+    thickness = 50.0
+    toroidal_extent = 90.0
+    sample_mod = 6
+
+    stellarator_obj.construct_magnets_from_filaments(
+        coils_file, width, thickness, toroidal_extent, sample_mod=sample_mod
+    )
+
+    step_filename_exp = "magnet_set.step"
+
+    stellarator_obj.export_magnets(step_filename=step_filename_exp)
+
+
+def create_ivb_cad_magnets_from_cad(stellarator_obj):
+    toroidal_angles = [0.0, 5.0, 10.0, 15.0]
+    poloidal_angles = [0.0, 120.0, 240.0, 360.0]
+    wall_s = 1.08
+    component_name_exp = "component"
+    radial_build_dict = {
+        component_name_exp: {
+            "thickness_matrix": np.ones(
+                (len(toroidal_angles), len(poloidal_angles))
+            )
+            * 10
+        }
+    }
+    num_ribs = 11
```

Comment:
This is reused across multiple tests. Can we define once? Maybe a test fixture?

Reply:
These `create_...` functions aren't tests, but since it's repeated across multiple functions, I agree it's probably best to define just once